components/style/error_reporting.rs

```diff
@@ -27,11 +28,13 @@ pub trait ParseErrorReporter {
 /// TODO(emilio): The name of this reporter is a lie, and should be renamed!
 pub struct StdoutErrorReporter;
 impl ParseErrorReporter for StdoutErrorReporter {
-    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str) {
+    fn report_error(&self, input: &mut Parser, position: SourcePosition, message: &str,
+        url: Option<&ServoUrl>) {
```

Comment:
I could be wrong, but I believe `url` will never be `None` so we should change `Option<&ServoUrl>` to `&ServoUrl` and remove any unneeded unwraps.